app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.use(express.static("frontend"));
+
+app.get("/", function (req, res) {
```

Comment:
express.static("frontend") will look at your index.html and make an endpoint for "/"

What happens if you also then later on define another thing it should do for the endpoint "/"? (Like you do here)